default/generated/00-discovery/0.yaml

```diff
@@ -30,6 +31,7 @@
   description: "Hardcoded IP Address"
   labels:
   - konveyor.io/target=cloud-readiness
+  - konveyor.io/target=discovery
```

Comment:
Let's not add a discovery to this rule; instead, we can move it to the cloud-readiness folder.